harness/determined/core/_context.py

```diff
@@ -39,20 +39,26 @@ def __init__(
         preempt: Optional[core.PreemptContext] = None,
         train: Optional[core.TrainContext] = None,
         searcher: Optional[core.SearcherContext] = None,
+        tensorboard_manager: Optional[tensorboard.TensorboardManager] = None,
     ) -> None:
         self.checkpoint = checkpoint
         self.distributed = distributed or core.DummyDistributedContext()
         self.preempt = preempt or core.DummyPreemptContext(self.distributed)
         self.train = train or core.DummyTrainContext()
         self.searcher = searcher or core.DummySearcherContext(self.distributed)
+        self.tensorboard_manager = tensorboard_manager
```

Comment:
core.Context is a user-facing object, and all these other context objects are public-facing namespaces.

But I don't think we want `tensorboard_manager` to be a user-facing call, so this must have an underscore prefix.